app/assets/stylesheets/donations.scss

```diff
@@ -3,3 +3,149 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .stripe-button-el { display: none !important }
+
+
+img.card {
+	margin-top: 10px; 
+	height: 11%; 
+	width: 11%;
+	display: inline-block;
+}
+
+img.cardstripe {
+	float: left ;
+	margin-top: 10px; 
+	height: 27%; 
+	width: 27%;
+	display: inline-block;
+}
+
+img.lock{
+	height: 5%; 
+	width: 5%;
+	display: inline-block;
+}
+
+#div_5dac_0{
```

Comment:
this is a really bad way to use IDs but since it's the last minute it's fine I guess.

Reply:
one of these styles also extends the homepage bar. I know because when I just deleted the inline styles, that fixed the homepage bar.